src/clj_sqs_extended/internal/receive.clj

```diff
@@ -66,31 +65,22 @@
 
 (defn- error-might-be-recovered-by-restarting
```

Comment:
rename to `error-might-be-recovered-by-restarting?` to clearly indicate a predicate fn